lib/core/src/client/preview/start.js

```diff
@@ -349,8 +349,13 @@ export default function start(render, { decorateStory } = {}) {
       Object.keys(exports).forEach(key => {
         if (isExportStory(key, meta)) {
           const storyFn = exports[key];
-          const { name, parameters } = storyFn.story || {};
-          kind.add(name || key, storyFn, parameters);
+          const { name, parameters, decorators } = storyFn.story || {};
+          if (parameters && parameters.decorators) {
+            deprecate(() => {},
```

Comment:
I think with these `deprecate` calls you want to create them only once so they only get logged once, no?